src/AzureIoTHub.Portal/Shared/Models/v1.0/IoTEdgeModule.cs

```diff
@@ -30,11 +31,16 @@ public class IoTEdgeModule
         /// <summary>
         /// The module environment variables.
         /// </summary>
-        public Dictionary<string, string> EnvironmentVariables { get; } = new Dictionary<string, string>();
+        public List<IoTEdgeModuleEnvironmentVariable> EnvironmentVariables { get; set; } = new List<IoTEdgeModuleEnvironmentVariable>();
```

Comment:
What is the reason to remove Dictionary here ?

Reply:
There were issues with editing keys in the Dictionary type (and mapping them to the editing table in the dialog), so I created new models (EnvironmentVariable, TwinSettings and Commands) and used List of theses models instead.

Reply:
ok